pkg/cloud/network.go

```diff
@@ -17,26 +17,13 @@ limitations under the License.
 package cloud
 
 import (
-	"strconv"
-	"strings"
-
-	"github.com/apache/cloudstack-go/v2/cloudstack"
 	capcv1 "github.com/aws/cluster-api-provider-cloudstack/api/v1beta1"
 	"github.com/hashicorp/go-multierror"
 	"github.com/pkg/errors"
 )
 
 type NetworkIface interface {
-	ResolveNetworkStatuses(*capcv1.CloudStackCluster) error
-	ResolveNetwork(*capcv1.CloudStackCluster, *capcv1.Network) error
-	CreateIsolatedNetwork(*capcv1.CloudStackCluster) error
-	OpenFirewallRules(networkID string) error
-	FetchPublicIP(*capcv1.CloudStackCluster) (*cloudstack.PublicIpAddress, error)
-	ResolveLoadBalancerRuleDetails(*capcv1.CloudStackCluster) error
-	GetOrCreateLoadBalancerRule(*capcv1.CloudStackCluster) error
-	GetOrCreateIsolatedNetwork(*capcv1.CloudStackCluster) error
-	AssociatePublicIPAddress(*capcv1.CloudStackCluster) error
-	DeleteNetwork(capcv1.Network) error
+	ResolveNetwork(*capcv1.Network) error
```

Comment:
Since outside of the Isolated Network case, all we do is fetch info, this a now a pretty small thing.